src/arm/overlays/BBAI-SPI0_1.dts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) 2020 Deepak Khatri <deepaklorkhatri7@gmail.com>
+ * https://lorforlinux.github.io/GSoC2020_BeagleBoard.org/
+ *
+ * See Cape Interface Spec page for more info on Bone Buses
+ * https://elinux.org/Beagleboard:BeagleBone_cape_interface_spec
+ *
+ * BBAI Virtual cape for /dev/bone/spi/0.1
```

Comment:
Please add 0.1 to the wiki. It is not currently there.

Reply:
I was just waiting for you to merge it updating...